corehq/apps/analytics/templates/analytics/initial/gtm.html

```diff
@@ -11,3 +11,15 @@
 {% if ANALYTICS_CONFIG.HQ_INSTANCE %}
   {% initial_analytics_data 'gtm.hqInstance' ANALYTICS_CONFIG.HQ_INSTANCE %}
 {% endif %}
+{% if request.project %}
+  {% if request.project.is_test == 'true' %}
+    {% initial_analytics_data 'gtm.isTestDomain' 'yes' %}
+  {% elif request.project.is_test == 'false' %}
+    {% initial_analytics_data 'gtm.isTestDomain' 'no' %}
+  {% endif %}
+  {% initial_analytics_data 'gtm.isDomainActive' request.project.recent_submissions %}
```

Comment:
Does `request.project.recent_submissions` return a true/false or yes/no value? If it's the former then you might have to apply the same logic as used with `request.project.is_test` to format the value to return yes/no instead of true/false.

Reply:
`request.project.recent_submissions` returns a boolean True/False as opposed to string returned by `request.project.is_test`, so it is converted to yes/no in the JS [here](https://github.com/dimagi/commcare-hq/pull/34862/commits/49c6b6cbacd577331e8aa7c189511e4c3b6ad6a3#diff-2e828abac1f9c31907a4c32949a8a3ffa0ab9f23da41b48fd27bb792d806eadeR59).

Reply:
Ah, thanks for the clarification. I thought that was simply looking at whether the value was 'no' or 'yes'. If we convert it in the JS, do we then still need to have the conversion happen in the HTML for `request.project.is_test`?

Reply:
> If we convert it in the JS, do we then still need to have the conversion happen in the HTML for request.project.is_test?

Good question. Yes because the `request.project.is_test` returns string values ('true', 'false' , 'none') and the JS API that fetches data from HTML works on boolean values which considers `false` as `yes`.